sourcegraph/cody
# fix: reset current chat after clearing history

fix: resetting transcript and session on clear history

Current clearing chat history does not reset current chat sessions, so if you use a command after clearing the chat session, you will still see the last chat session when a new session is expected

This PR includes changes to address the above issue:
- Reset the transcript when clearing chat history
- Call clearAndRestartSession() after clearing chat history to reset the current chat session
- Send updated history after saving to storage
- Add test to cover the updated behavior

## Test plan



1. click button to clear history
2. Use a command from your right click menu
3. The command output should show up in a new chat session without old convo
